dora/core/server/master/src/main/java/alluxio/master/job/MoveJob.java

```diff
@@ -215,13 +215,15 @@ public void setVerificationEnabled(boolean enableVerification) {
    */
   @Override
   public void failJob(AlluxioRuntimeException reason) {
+    LOG.info("Move Job {} fails", mJobId);
```

Comment:
do you mind put more information here like the whole job status

Reply:
I put the job description here.